torch/csrc/distributed/rpc/tensorpipe_agent.cpp

```diff
@@ -110,6 +113,48 @@ std::vector<c10::Stream> getCurrentStreamsForDevices(
   return streams;
 }
 
+std::vector<c10::Device> getDevicesOfTensors(
+    const std::vector<torch::Tensor>& tensors) {
+  c10::optional<c10::impl::VirtualGuardImpl> impl;
+  size_t deviceCount = 0;
+  std::vector<bool> indexBitSet;
```

Comment:
curious about this var name. It seems to be neither `bit` nor `set`? Or is it possible to use a real `std::bitset` here?

Reply:
This is probably not the best capitalization, but the name was meant to be "a bitset of indices". It is conceptually identical to `std::bitset`, but if you notice `std::bitset` requires a _compile-time_ size. The doc for `std::bitset` says:
> If the size of the bitset is not known at compile time, std::vector<bool> may be used.

Hope this clarifies. I'll fix the capitalization.